util/logger-service/src/lib.rs

```diff
@@ -482,6 +482,27 @@ pub fn init(env_opt: Option<&str>, config: Config) -> Result<LoggerInitGuard, Se
     })
 }
 
+/// Initializes the [SilentLogger](struct.SilentLogger.html).
+pub fn init_silent() -> Result<LoggerInitGuard, SetLoggerError> {
+    log::set_boxed_logger(Box::new(SilentLogger)).map(|_| LoggerInitGuard)
+}
+
+/// The SilentLogger which implements [log::Log].
+///
+/// Silent logger that does nothing.
+#[derive(Debug)]
+pub struct SilentLogger;
```

Comment:
I think this `SilentLogger` is unnecessary.

Set the follow field to `false` is enough:
https://github.com/nervosnetwork/ckb/blob/dc3cf8d069bfbe8b9cbe0c0095f95aab70b71ca2/util/logger-config/src/lib.rs#L37-L38

Reply:
On the contrary, I think normal `Logger` is unnecessary when we just want silent logging,  the `Logger` is so bloated with a complex logical background writer.